### DISCUSSION TITLE
filtering the doublets and ArchR's and Signac filtering criteria

### DISCUSSION THREAD

**Discussion by tanasa** [GreenleafLab/ArchR#1413] [General]
Hi Ryan, 

It would be great to have your very well informed on these two questions below :

1. I would like to better understand the way in which the doublet score is computed in ArchR in contrast with just considering the number of fragments per cell.  

We do have samples where the median number for fragments per cell is very high i.e. > 70 000, and I am thinking that there might have been issues with the dissociation protocol.  In the published datasets, the median number of fragments per cell is ~ 15 000.

2. Why ArchR filtering criteria are different from Signac's, except TSS enrichment ?

Thanks a lot, 
 
~ Bogdan

**Comment by rcorces** [comment]
1. See the manuscript main text and methods section- https://www.nature.com/articles/s41588-021-00790-6
2. We have not found the need / utility for additional QC metrics.
